lib/signs_ui/signs/sign.ex

```diff
@@ -147,6 +160,14 @@ defmodule SignsUI.Signs.Sign do
     }
   end
 
+  @spec expiration_to_iso8601(expires_on()) :: String.t() | nil
+  defp expiration_to_iso8601(expires) do
```

Comment:
Maybe two function clauses, both for brevity and for a bit of struct-assertion?

```ex
defp expiration_to_iso8601(%DateTime{} = dt), do: DateTime.to_iso8601(dt)
defp expiration_to_iso8601(nil), do: nil
```

Reply:
Done